index.css

```diff
@@ -635,29 +660,26 @@ body {
 
 .form {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   align-items: flex-start;
   gap: 58px;
   width: 90%;
-  margin: 0 5% 0 5%;
+  margin: 120px 5% 0 5%;
 
   /* no unit */
-  top: 6338px;
 }
 
 .form_item {
   display: flex;
   flex-direction: column;
   align-items: center;
   gap: 24px;
-  flex: none;
   flex-grow: 0;
   align-self: center;
 }
 
 .form_item1 {
   order: 0;
-  margin-top: 120px;
 }
```

Comment:
- Kindly, stick to the figma design as match as possible, and align the text correctly as your previous code reviewer requested you.👌👌



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206780450-96426274-cedc-4a06-bd1d-ec2669be0427.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206780456-1fb79746-6ac5-40ca-b3eb-8611c4f554bd.png" width="200">|


😎️